src/schemas/ev42/ev42_rw.cxx

```diff
@@ -83,26 +83,34 @@ void writer::init_impl(std::string const &sourcename, hid_t hdf_group,
                        Msg msg) {
   LOG(7, "ev42::init_impl");
 
+  hsize_t chunk_n_elements = 1;
+
   if (config_file) {
     if (auto x = get_int(config_file, "nexus.indices.index_every_kb")) {
       index_every_bytes = (int)x * 1024;
```

Comment:
There is a signed to unsigned cast here which should be explicit if intentional, and the C-style could be removed in the process.